docs/GETTING_STARTED_GUIDE.md

```diff
@@ -117,7 +117,7 @@ Karavi Authorization has a subset of configuration parameters that can be update
 
 | Parameter | Type | Default | Description |
 | --------- | ---- | ------- | ----------- |
-| web.sidecarproxyaddr | String |"127.0.0.1:5000/sidecar-proxy:latest" |Registry address of the Karavi Authorization sidecar-proxy |
+| web.sidecarproxyaddr | String |"127.0.0.1:5000/sidecar-proxy:1.0.0" |Registry address of the Karavi Authorization sidecar-proxy |
```

Comment:
No changes required here.  Content has moved to the common doc site so please make changes there instead

Reply:
removed